game-core/src/main/java/games/strategy/triplea/attachments/TriggerAttachment.java

```diff
@@ -1514,9 +1556,10 @@ public static void triggerRelationshipTypePropertyChange(final Set<TriggerAttach
               change.add(
                   ChangeFactory.attachmentPropertyChange(attachment, newValue, property.getFirst(), clearFirst));
             }
-            bridge.getHistoryWriter().startEvent(MyFormatter.attachmentNameToText(t.getName()) + ": Setting "
-                + property.getFirst() + (newValue.length() > 0 ? " to " + newValue : " cleared ") + " for "
-                + t.getRelationshipTypeAttachmentName().getSecond() + " attached to " + relationshipType.getName());
+            bridge.getHistoryWriter()
+                .startEvent(MyFormatter.attachmentNameToText(t.getName()) + ": Setting " + property.getFirst()
+                    + (newValue.length() > 0 ? " to " + newValue : " cleared ") + " for "
+                    + t.getRelationshipTypeAttachmentName().getSecond() + " attached to " + relationshipType.getName());
```

Comment:
`String.format()` would help here as well, even though this line doesn't really have anything to do with the rest of the refatoring